templates/ts-fastify-mongodb-backend/src/index.ts

```diff
@@ -45,6 +45,7 @@ async function start() {
 
   app.register((mercurius as any), {
     schema,
+    resolvers,
```

Comment:
I think `resolvers` should include `noteResolvers` as in https://github.com/aerogear/graphback/blob/12d588718dacaecb5cc1d3ed8005ac4a949b82a4/templates/ts-fastify-mongodb-backend/src/index.ts#L40